geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DeltaWriter.scala

```diff
@@ -217,6 +219,18 @@ object DeltaWriter extends StrictLogging {
       SimpleFeatureOrdering.nullCompare(y._1.asInstanceOf[Comparable[Any]], x._1)
   }
 
+  @deprecated("Use reduce without an allocator")
```

Comment:
Thanks for getting this change in.  Looks like my branch showing that the allocator was unused had a benefit:).

Reply:
the allocator is still used, but now it's wrapped in the close() lifecycle of the result.